commands/team_users_e2e_test.go

```diff
@@ -0,0 +1,139 @@
+package commands
```

Comment:
You have to add the license header here :) (Just copy it from any other file in the project)